<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1142</width>
    <height>775</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Netperf-Wrapper GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="viewArea">
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1142</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="action_Close_tab"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="action_Plot_selector"/>
    <addaction name="action_Settings"/>
    <addaction name="action_Metadata"/>
    <addaction name="separator"/>
    <addaction name="actionNextTab"/>
    <addaction name="actionPrevTab"/>
   </widget>
   <widget class="QMenu" name="menu_Data">
    <property name="title">
     <string>&amp;Data</string>
    </property>
    <addaction name="actionLoadExtra"/>
    <addaction name="actionOtherExtra"/>
    <addaction name="actionClearExtra"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Data"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="plotDock">
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Select plot</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QListView" name="plotView">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="settingsDock">
   <property name="windowTitle">
    <string>Plot settings</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QCheckBox" name="checkLegend">
       <property name="toolTip">
        <string>Draw plot legend?</string>
       </property>
       <property name="statusTip">
        <string>Draw plot legend?</string>
       </property>
       <property name="text">
        <string>Legend</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="checkAnnotation">
       <property name="toolTip">
        <string>Annotate plot (at the bottom)?</string>
       </property>
       <property name="statusTip">
        <string>Annotate plot (at the bottom)?</string>
       </property>
       <property name="text">
        <string>Annotation</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="checkZeroY">
       <property name="toolTip">
        <string>Always start Y-axis at 0?</string>
       </property>
       <property name="statusTip">
        <string>Always start Y-axis at 0?</string>
       </property>
       <property name="text">
        <string>Zero Y-axis</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="checkDisableLog">
       <property name="toolTip">
        <string>Disable automatic log scaling of axes?</string>
       </property>
       <property name="statusTip">
        <string>Disable automatic log scaling of axes?</string>
       </property>
       <property name="text">
        <string>Disable log scale</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="checkTitle">
       <property name="toolTip">
        <string>Draw plot title?</string>
       </property>
       <property name="statusTip">
        <string>Draw plot title?</string>
       </property>
       <property name="text">
        <string>Title</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="checkScaleMode">
       <property name="toolTip">
        <string>Use extra data files for axis scaling instaed of plotting them?</string>
       </property>
       <property name="statusTip">
        <string>Use extra data files for axis scaling instaed of plotting them?</string>
       </property>
       <property name="text">
        <string>Scale mode</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="metadataDock">
   <property name="windowTitle">
    <string>Metadata</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTreeView" name="metadataView">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open data file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Close_tab">
   <property name="text">
    <string>&amp;Close tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="action_Plot_selector">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Plot selector</string>
   </property>
  </action>
  <action name="action_Settings">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Settings</string>
   </property>
  </action>
  <action name="action_Metadata">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Metadata</string>
   </property>
  </action>
  <action name="actionLoadExtra">
   <property name="text">
    <string>&amp;Load additional data file</string>
   </property>
  </action>
  <action name="actionOtherExtra">
   <property name="text">
    <string>&amp;Add other open data files</string>
   </property>
  </action>
  <action name="actionClearExtra">
   <property name="text">
    <string>&amp;Clear extra data files</string>
   </property>
  </action>
  <action name="actionNextTab">
   <property name="text">
    <string>&amp;Next tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Tab</string>
   </property>
  </action>
  <action name="actionPrevTab">
   <property name="text">
    <string>P&amp;revious tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Backtab</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>action_Quit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>570</x>
     <y>387</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Metadata</sender>
   <signal>toggled(bool)</signal>
   <receiver>metadataDock</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>1007</x>
     <y>671</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Settings</sender>
   <signal>toggled(bool)</signal>
   <receiver>settingsDock</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>1007</x>
     <y>561</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Plot_selector</sender>
   <signal>toggled(bool)</signal>
   <receiver>plotDock</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>1007</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
